kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/ParserHelper.java

```diff
@@ -247,6 +247,17 @@ public static String getOriginalText(ParserRuleContext ctx) {
         return ctx.getStart().getInputStream().getText( interval );
     }
 
+    /**
+     * Sanitizes a number of the form: <code>123e+10</code> and turns it into a valid identifier
+     */
+    public static String toNumericIdentifier(String number) {
+        // FIXME quick & dirty sanitization routine
+        return "K_" + number
+                .replace(".", "$point$")
```

Comment:
`replace(".", escapeIdentifier(".")`

Reply:
^ suggestion please

Reply:
so `escapeIdentifier(number)` ?

Reply:
yeah even simpler, dunno why I didn't do that already before.